docs/src/sources-files.md

```diff
@@ -89,48 +81,116 @@ pmtiles:
     tiles: s3://bucket/path/to/tiles.pmtiles
 ```
 
-The available options depend on the underlying source:
-
-{{#tabs }}
-{{#tab name="local file system" }}
-Files don't currently support additional options.
-{{#endtab }}
-{{#tab name="Http(s)" }}
+### Available options:
 
 {{#include pmtiles/client.md}}
 
 {{#endtab }}
 {{#tab name="Amazon S3" }}
 
+> [!IMPORTANT]
+> Even though we name this section `Amazon S3`, it also works with other providers that support the S3 API, such as [MinIO](https://www.min.io/), [Ceph](https://docs.ceph.com/en/latest/radosgw/s3/), [Cloudflare R2](https://developers.cloudflare.com/r2/), [hetzner object storage](https://www.hetzner.com/de/storage/object-storage/) and many more.
```

Comment:
I am calling out a few options here, I think it it is hopefully representative sample of what I think people not on S3 use, but who knows.